### PR TITLE
Adjust code for Accessibility

### PR DESCRIPTION
**Pull request by Linushaddai99**
In the sixth milestone of your portfolio website, we need to:

evaluate our website against accessibility best practices and implement any necessary changes to ensure that our website is accessible.
Here are the changes we made:

Page titles --------------------------------------------------------- no issues found
Image text alternatives** (alt text) Changed

[Changed No 1](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R15)
[Changed No 2](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R24-R27)
[Changed No 3](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R56)
[Changed No 4](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R65)
[Changed No 5](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R75)
[Changed No 6](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R99)
[Changed No 7](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R123)
[Changed No 8](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R147)
[Changed No 9](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R194)
[Changed No 10](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R202)

Text headings --------------------------------------------------------- no issues found
Color contrast --------------------------------------------------------- issues found

To match the Figma design and to solve the problem of Color contrast, we changed the color to the nearest value, to meet the accessibility standard of contrast ratio which is 4:5:1

[Changed No 1](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR112)
[Changed No 2](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR179)
[Changed No 3](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR194)
[Changed No 4](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR302)
[Changed No 5](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR330)

Resize --------------------------------------------------------- no issues found
Interaction (labels are optional but recommended as voiceover tools can read them, and should be set as invisible to match the Figma design) --------------------------------------------------------- issues found

Changed but to match the Figma Design added a font size of 0; [View Here](https://github.com/Linushaddai99/My-Portfiolio/pull/8/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR342-R345)

Multimedia --------------------------------------------------------- no issues found
The basic structure of the page --------------------------------------------------------- no issues found

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ In this project, I was able to build my personal portfolio . The goal of this pr
 - Twitter: [@ShaddaiLinus](https://twitter.com/ShaddaiLinus)
 - Linkedin: [@ShaddaiLinus](https://www.linkedin.com/in/linusshaddai/)
```
Comment (andgarzonmal) [inline README.md:33]: - [ ] Hello 👋 please remember that according to the project requirements 👨‍🏫 you should check all of the items listed in the article [Easy Checks – A First Review of Web Accessibility](https://www.w3.org/WAI/test-evaluate/preliminary/) and make a report according to your findings 

image

Kyndly add your report to the description of your pull request.

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md (hunk 1 above)
Comment (Linushaddai99) [reply README.md:33]: I'm sorry but I'm not sure I understand what you want us to do.

**Review by Reem-lab — APPROVED** [review]
# Hi Team,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the accessibilty✅
📌 Good commit messages ✅

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**